data/tsv/bur_mymr_phonemic.tsv

```diff
@@ -1425,7 +1425,6 @@
 ထမင်းချဉ်	tʰ ə m ɪ́ ɴ d͡ʑ ɪ̀ ɴ
 ထမင်းဆိုင်	tʰ ə m ɪ́ ɴ z à ɪ ɴ
 ထမင်းသုပ်	tʰ a̰ m ɪ́ ɴ θ o ʊ ʔ
-ထမင်း‌ကြော်	tʰ ə m ɪ́ ɴ‌ d͡ʑ ɔ̀
```

Comment:
I don't understand why these words are being excluded.

Reply:
This gets excluded because of the ZWJ/ZWNJ which are common to all Brahmic scripts. We need solution for this as well.

```
['MYANMAR LETTER THA', 'MYANMAR LETTER MA', 'MYANMAR LETTER NGA', 'MYANMAR SIGN ASAT', 'MYANMAR SIGN VISARGA', 'ZERO WIDTH NON-JOINER', 'MYANMAR LETTER KA', 'MYANMAR CONSONANT SIGN MEDIAL RA', 'MYANMAR VOWEL SIGN E']
```

Reply:
I suppose a quick (and temporary) solution would be to extend our `regex_string` to include `\u200C\u200D`? I think in the future we'll only want to extend `regex_string`'s coverage uniquely for each language but extending it for all languages now is a bit easier.